pkg/repstream/streampb/stream.proto

```diff
@@ -66,15 +66,25 @@ message StreamPartitionSpec {
     google.protobuf.Duration min_checkpoint_frequency = 2
        [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
 
-    // Controls batch size in bytes.
+    // Controls the rangefeed batch size in bytes.
```

Comment:
I'm not sure this is actually the range-feed batch size? Rather it is the size of batches we send over pgwire

Reply:
whoops. you're right. I was thinking about rangefeedcaches and not rangefeeds.